lib/model/message.dart

```diff
@@ -132,6 +163,55 @@ class MessageStoreImpl extends PerAccountStoreBase with MessageStore {
     }
   }
 
+  @override
+  bool? getEditMessageErrorStatus(int messageId) =>
+    _editMessageRequests[messageId]?.hasError;
+  final Map<int, _EditMessageRequestStatus> _editMessageRequests = {};
```

Comment:
nit: this field is a bit more than the backing for this getter-like method:

```suggestion
  @override
  bool? getEditMessageErrorStatus(int messageId) =>
    _editMessageRequests[messageId]?.hasError;

  final Map<int, _EditMessageRequestStatus> _editMessageRequests = {};
```

because in particular it has more data than that exposes.